code/parse/sexp.cpp

```diff
@@ -5010,6 +5010,12 @@ int sexp_is_destroyed(int n, fix *latest_time)
 			if ( latest_time && (time > *latest_time) )
 				*latest_time = time;
 		} else {
+			// If a previous SEXP already had an empty wing then this code would expose the internal value as the
+			// directive count. Instead, we reset the count to zero here to make sure that the wing or ship count is
+			// correct.
+			if (Directive_count == DIRECTIVE_WING_ZERO) {
```

Comment:
If this problem really warrants a warning in the SEXP help, then should there be a debug log statement here too?

Reply:
Oh right, good thinking.  Yes, we use boolean-guarded log prints elsewhere, so let's use one here too.

(How about resetting the boolean each time a mission is restarted, perhaps in the sexp initialization?)

Reply:
I added the debug log print with the static boolean guard. Resetting the value when a mission is loaded would require some ugly extern references to such a variable which I would like to avoid so I kept it simple.

Reply:
Okay fair enough.